dev/tasks/r/github.nightly.yml

```diff
@@ -0,0 +1,424 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+{% import 'macros.jinja' as macros with context %}
+# root of the R nightly repo where the first dir is the mount point used by davfs2 ("nightly")
+{% set r_root = "nightly/r" %}
+{% set webdav_domain = "https://nightly.wujciak.de:8080" %}
+{% set repo_domain = "https://nightly.wujciak.de/r" %}
+# This allows us to set a custom version via param:
+# crossbow submit --param custom_version=8.5.3 r-nightly-packages
+# if the param is unset defaults to the usual Ymd naming scheme
+{%set version = custom_version|default('\\2.\'\"$(date +%Y%m%d)\"\'') %}
+
+{{ macros.github_header() }}
+
+env:
+# prevent upload of R packages or libarrow binaries via param (e.g. for release)
+# crossbow submit --param upload_pkg=false --param upload_libarrow=false r-nightly-packages
+  UPLOAD_PKG: {{ upload_pkg|default("true") }}
+  UPLOAD_LIBARROW: {{ upload_libarrow|default("true")}}
+jobs:
+  source:
+    name: Source Package
+    runs-on: ubuntu-latest
+    # Block submissions via PR unless from a branch within apache/arrow e.g. for release 
+    if: '{{ arrow.github_repo }}' == 'apache/arrow'

```

Comment:
This will prevent this workflow from running should someone trigger it from a PR (e.g. via -g nightly). This also means that if we want to create the binaries for the release we need to do it from an apache/arrow branch and not a fork. 

It could also be nice to allow org members (e.g. you for release) to start this from their fork PRs but afaik the actor information is not available within the job render process right @kszucs ?

Reply:
It can now be run from PRs